climada/hazard/trop_cyclone.py

```diff
@@ -336,46 +336,93 @@ def from_tracks(
 
     def apply_climate_scenario_knu(
         self,
-        ref_year: int = 2050,
-        rcp_scenario: int = 45
+        pct: int = 2,
```

Comment:
In my opinion:

1. Any user calling this function should explicitly specify for which future year and rcp scenario they are targetting. Most likely, the baseline years is also something you would like to force the user to provide, although I can see why a default could also be considered for that argument. Input parameter `pct`, that's for sure, has a clear default: the median.
2. Rather than using an argument named `pct` which is actually an encoded value, I suggest naming it `percentile` and its value to be the actual percentile (default: 50), with a quick check at the beginning that it is one of the allowed values (if not, raise an error). Or is the encoding because value 0 refer sometimes to 5 or 10 and value 4 sometimes to 90 or 95? In that case, can we make this consistent by using 10/90 and explain in the docstring that the 10/90 actually refer to 5/95 for the cat 0-5 frequencies?

Therefore I suggest to use the following function definition:
```
    def apply_climate_scenario_knu(
        self,
        ref_year: int,
        rcp_scenario: int,
        baseline: tuple,
        percentile: int = 50,
    ):
```
If you go for this, the docstring will have to be adjusted accordingly (and unit tests as well I presume).

Reply:
Alternatively, if `pct` (which I'd rename anyway) cannot be 10/25/50 etc, then perhaps we could use a string encoding such as 'median', 'IQR-', 'IQR+', 'high', 'low'. Just a spontaneous idea here.